tests/test_room_resource/test_assign_resource.py

```diff
@@ -0,0 +1,80 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.token.token_fixture import ADMIN_TOKEN
+from fixtures.room.assign_resource_fixture import (
+    assign_resource_mutation,
+    assign_resource_mutation_response,
+    assign_resource_non_existent_room,
+    assign_non_existent_resource_id,
+    assign_excess_quantity,
+    assign_quantity_less_than_one,
+    query_string
+    )
+
+import sys
+import os
+sys.path.append(os.getcwd())
+
+
+class TestAssignResource(BaseTestCase):
+
+    def test_assign_resource(self):
+        """
+            Test an admin can assign a resource to a room
+        """
+        CommonTestCases.admin_token_assert_equal(
+           self,
+           assign_resource_mutation,
+           assign_resource_mutation_response,
+        )
+
+    def test_assign_non_existent_resource(self):
+        """
+            Test assigning a resource that does not exist
```

Comment:
Are we testing whether this is possible or not? Kindly indicate so for all the tests below.